types/messages.go

```diff
@@ -138,103 +130,206 @@ func (msg *SSVMessage) Decode(data []byte) error {
 
 // SignedSSVMessage is the main message passed within the SSV network. It encapsulates the SSVMessage structure and a signature
 type SignedSSVMessage struct {
-	Signature  [256]byte // Created by the operator's network key
-	OperatorID OperatorID
-	Data       []byte
+	Signatures  [][]byte     `ssz-max:"13,256"` // Created by the operators' key
+	OperatorIDs []OperatorID `ssz-max:"13"`
+	SSVMessage  *SSVMessage
+	// Full data max value is ConsensusData max value ~= 2^8 + 8 + 2^20 + 2^22 = 5243144
+	FullData []byte `ssz-max:"5243144"`
 }
 
-// GetOperatorID returns the sender operator ID
-func (msg *SignedSSVMessage) GetOperatorID() OperatorID {
-	return msg.OperatorID
+// GetOperatorIDs returns the sender operator ID
+func (msg *SignedSSVMessage) GetOperatorIDs() []OperatorID {
+	return msg.OperatorIDs
 }
 
 // GetSignature returns the signature of the OperatorID over Data
-func (msg *SignedSSVMessage) GetSignature() [256]byte {
-	return msg.Signature
+func (msg *SignedSSVMessage) GetSignature() [][]byte {
```

Comment:
the name is now tricky imo should be called GetSignatures

Reply:
I notice we don't even call this method along another one so I am simply deleting

Reply:
yeah... not sure if we do it now